packages/core/docs/integrations/adyen.md

```diff
@@ -236,3 +238,11 @@ For more information, see the 6th step of the [Adyen's installation guide](./ady
 
 ### Klarna Pay Later does not work for United States
 Klarna Pay Later is not supported in the United States. However, sometimes it is added when you enable Klarna in Adyen's dashboard. If you have this problem, contact Adyen's support to remove it.
+
+### Nothing happens after 3DS2 Auth in one environment but works in a different environment
+There might be a situation when you can finish 3DS2 Auth in the local environment. But it is impossible on staging. It simply means you've provided `http://localhost:3000` as an `origin` in the `middleware.config.js` instead of your staging's URL. 
+
+It can work properly only on provided `origin`.
+
+### Structure of DetailsRequest contains the following unknown fields...
+Update version of the both extension module and notification module. Just modify tag to the newest version in `extension.Dockerfile` and `notification.Dockerfile`.
```

Comment:
```suggestion

Update extension and notification modules to the newest version by updating the tag in `extension.Dockerfile` and `notification.Dockerfile`.

```

Reply:
I suggest being more precise with the work `newest` since it might be different in the future. If you think it's okay, we can leave it.